toolkit/src/main/cpp/export/meta_toolkit_export_plugin.cpp

```diff
@@ -88,15 +111,28 @@ Dictionary MetaToolkitExportPlugin::_get_export_options_overrides(
 	}
 
 	// Check if this plugin is enabled
-	if (!_get_bool_option("meta_toolkit/enable_meta_toolkit")) {
+	if (!_is_plugin_enabled()) {
 		return overrides;
 	}
 
 	// Gradle build overrides
-	overrides["gradle_build/use_gradle_build"] = true;
 	overrides["gradle_build/export_format"] = 0; // apk
 	overrides["gradle_build/min_sdk"] = "29"; // Android 10
 	overrides["gradle_build/target_sdk"] = "34"; // Android 14
```

Comment:
I think these need to be moved into the `if (!is_mobile_editor)` below too, because otherwise I get these two errors:

```
"Min SDK" can only be overidden when "Use Gradle Build" is enabled.
"Target SDK" can only be overidden when "Use Gradle Build" is enabled.
```